src/lib/y2security/security_policies/bootloader_password_rule.rb

```diff
@@ -18,17 +18,22 @@
 # find current contact information at www.suse.com.
 
 require "y2security/security_policies/rule"
+require "y2storage/arch"
 
 module Y2Security
   module SecurityPolicies
     #  Run to check whether grub2 is password-protected and menu editing is restricted
-    #  (SLES-15-010200).
     class BootloaderPasswordRule < Rule
       def initialize
         textdomain "security"
 
+        # DISA STIG defines a rule for UEFI (SLES-15-010200) and another rule for non-UEFI
+        # (SLES-15-010190). The condition to check both rules in YaST is exactly the same, so let's
+        # simply adapt the rule ID according to the system type.
+        id = Y2Storage::Arch.new.efiboot? ? "SLES-15-010200" : "SLES-15-010190"
```

Comment:
btw in bootloader you can redefine type of bootloader to use non-efi on efi and vice versa, but usually it is not good idea ( I know that it was useful for some strange archs where medium cannot be efi booted, but still there is issue with efi manager which cannot access efi menu), so I would ignore it here.